Makefile

```diff
@@ -204,7 +204,7 @@ test-backend-unit:
 	SQLALCHEMY_DATABASE_URL=sqlite:////tmp/local.db \
 	MLFLOW_TRACKING_URI=http://localhost:8001 \
 	PYTHONPATH=../jobs:$$PYTHONPATH \
-	LUMIGATOR_SECRET_KEY=abcdefghijklmnopqrstuvwxyz123456 \
+	LUMIGATOR_SECRET_KEY=7yz2E+qwV3TCg4xHTlvXcYIO3PdifFkd1urv2F/u/5o= \
```

Comment:
I guess we cannot retrieve it from the conftest, or the other way around?

Reply:
Yeah, I get the point that we have this in makefile and also in conftest. I don't know the easy way to make it just in one place. 

On the other hand, as long as they're valid format keys, each of them can be anything/different.